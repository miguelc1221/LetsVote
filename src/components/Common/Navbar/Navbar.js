import React, { Component } from 'react';
import { Menu, Segment, Container } from 'semantic-ui-react'
import Login from '../../Auth/Login';
import './Navbar.css';

class Navbar extends Component {
  state = { activeItem: 'home' }

  handleItemClick = (e, { name }) => this.setState({ activeItem: name })

  render() {
    const { activeItem } = this.state
    return (
      <Segment className="navbar">
        <Container>
          <Menu inverted borderless className="navbar__container">
            <Menu.Header className="navbar__header">Let's Vote</Menu.Header>
            <Login />
          </Menu>
        </Container>
      </Segment>
    );
  }
}

export default Navbar;