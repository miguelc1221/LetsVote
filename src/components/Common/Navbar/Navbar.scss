.navbar {
  border-radius: 0 !important;
  background-color: #2B95D6 !important;
  font-weight: bold;
  margin-bottom: 0 !important;
  &__container {
    background-color: transparent !important;
    a {
      color: white !important;
    }
  }
  &__header {
    font-size: 25px;
    font-weight: bold;
    align-self: center;
    cursor: pointer;
    color: white !important;
  }
}