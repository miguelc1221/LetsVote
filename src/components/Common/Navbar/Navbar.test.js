import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Navbar } from './Navbar';

describe('Navbar Component', () => {
  it('should render as expected', () => {
    const component = shallow(<Navbar auth={{}} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  })
})