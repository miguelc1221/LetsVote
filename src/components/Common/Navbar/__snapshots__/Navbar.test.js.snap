exports[`Navbar Component should render as expected 1`] = `
<Segment
  className="navbar">
  <Container>
    <Menu
      borderless={true}
      className="navbar__container"
      inverted={true}>
      <MenuHeader
        as={[Function]}
        className="navbar__header"
        onClick={[Function]}
        to="/">
        Let\'s Vote
      </MenuHeader>
      <Connect(Login) />
    </Menu>
  </Container>
</Segment>
`;
